cmd/vendor/github.com/cloudflare/cfssl/revoke/revoke.go

```diff
@@ -31,17 +31,20 @@ type Revoke struct {
 	// status of a certificate (i.e. due to network failure) causes
 	// verification to fail (a hard failure).
 	HardFail bool
+	// HardFailLck is a Mutex locker for the HardFail
+	HardFailLck sync.Mutex
```

Comment:
I work on several PRs in parallel. Please refer to the main PR message. It is still in progress I will let you know when it will be merged.